### ISSUE TITLE
'ascii' codec can't encode character '\xe9' in position 1: ordinal not in range(128)

### ISSUE THREAD

**Issue by merlinxcy** [smicallef/spiderfoot#497]
spiderfoot master Latest commit 7388d4c on 2 Feb Error
```
'ascii' codec can't encode character '\xe9' in position 1: ordinal not in range(128)
Unhandled exception (UnicodeEncodeError) encountered during scan. Please report this as a bug: ['Traceback (most recent call last):\n', '  File "/spiderfoot/sfscan.py", line 151, in startScan\n    self.ts.sf.cachePut("internet_tlds", self.ts.config[\'_internettlds\'])\n', '  File "/spiderfoot/sflib.py", line 433, in cachePut\n    fp.write(line.decode(\'utf-8\') + \'\\n\')\n', "UnicodeEncodeError: 'ascii' codec can't encode character '\\xe9' in position 1: ordinal not in range(128)\n"]
```

![image](https://user-images.githubusercontent.com/18378246/76741710-f625e000-67aa-11ea-919b-08670a7ce9f7.png)

**Comment by bcoles** [comment]
Probably related to #482 which is an issue with caching.

I know @smicallef was looking into this, but can't remember if it's been fixed in the 3.1 branch yet.

**Comment by normaliok** [comment]
I have the same issue too
